clients/js/package.json

```diff
@@ -34,13 +34,13 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "@solana/web3.js": "2.0.0-rc.1"
+    "@solana/web3.js": "2.0.0"
```

Comment:
Is this how we do this? Not `^2`?

Reply:
You are right, there is a missing `^` - fixed!